src/carma-platform/parse_ros_bags.py

```diff
@@ -144,7 +276,7 @@ def get_vehicle_odometry(ros_bag_file, output_file, time_offset):
         )
 
         for message in messages:
-            cdasim_time_ms = (message.header.stamp - time_offset).to_sec() * 1_000
+            cdasim_time_ms = math.floor((message.header.stamp - time_offset).to_sec() * 1_000)
```

Comment:
Why are we using `floor()` here? Would it be better to use `round()`?

Reply:
I added round(), but I also added some extra check because sometimes the script is failing with error:
```
Traceback (most recent call last):
  File "parse_ros_bags.py", line 325, in <module>
    main()
  File "parse_ros_bags.py", line 317, in main
    221, args.ros_bag_file, args.csv_dir / "pedestrian_odometry.csv", time_offset
  File "parse_ros_bags.py", line 244, in get_carla_object_odometry
    cdasim_time_ms = round((message.header.stamp - time_offset).to_sec() * 1_000)
  File "/home/carma/.opt/Python-3.7.8/lib/python3.7/site-packages/genpy/rostime.py", line 257, in __sub__
    return self.__class__(self.secs - other.secs, self.nsecs - other.nsecs)
  File "/home/carma/.opt/Python-3.7.8/lib/python3.7/site-packages/genpy/rostime.py", line 208, in __init__
    raise TypeError("time values must be positive")
    ```